scripts/src/precheck/submission_test.py

```diff
@@ -0,0 +1,165 @@
+import pytest
+import responses
+import sys
+
+sys.path.append(".")
```

Comment:
Nit: I'm assuming we're doing this here so that we can reference submission in the following line, but this is a Python-ism that bugs me a bit. I'm reading this as "add the files in `.` as import names".

Normally, you would do `from precheck import submission`, and then reference the classes as `submission.Submission`, right?

Shouldn't we have access to that without manipulating the default import directories?

For what it's worth, I do recognize that we have this in other source files, which is what makes this a nit. Feel free to disregard.